Doc/faq/programming.rst

```diff
@@ -293,7 +298,7 @@ It's good practice if you import modules in the following order:
 
 1. standard library modules -- e.g. ``sys``, ``os``, ``getopt``, ``re``
 2. third-party library modules (anything installed in Python's site-packages
-   directory) -- e.g. mx.DateTime, ZODB, PIL.Image, etc.
+   directory) -- e.g. ``mx.DateTime``, ``ZODB``, ``PIL.Image``
```

Comment:
I used ` `` ` since the standard library modules used them above (and also dropped etc following suit), so if we're looking to change these then perhaps we should change the ones at point 1. as well

Reply:
Oh, I don't know how I missed that, sorry. In that case, I made a suggestion to fix those as well.